<launch>
<!--     <include file="$(find darknet_ros)/launch/darknet_ros.launch"/> -->

    <node pkg="ros_evo"  type="ros_rgbd"  name="ros_rgbd"  output="screen">
        <!-- <param name="sensor" value="MONO"/> -->
        <param name="sensor" value="RGBD"/>
        <param name="online" value="true" type="bool"/>
        <!-- <param name="rosbag" value="false" type="bool"/> -->
        <param name="rosbag" value="true" type="bool"/>
        <!-- <param name="gravity_aixs" value="2" type="int"/> -->
        <param name="gravity_aixs" value="3" type="int"/>

        <!--TUM 3 因为用了rosbag timestamp correct, 所以topic ... -->
        <param name="rgb_topic" value="/rgb/image_raw" type="string"/>
        <param name="depth_topic" value="/depth_to_rgb/image_raw" type="string"/>
        <param name="bbox_topic" value="/darknet_ros/bounding_boxes" type="string"/>
        <param name="yamlfile" value="TUM3.yaml" type="string"/>
    </node>

</launch>
